src/components/views/elements/MemberEventListSummary.js

```diff
@@ -144,87 +144,105 @@ module.exports = React.createClass({
         );
     },
 
-    render: function() {
-        let summary = null;
+    shouldComponentUpdate: function(nextProps, nextState) {
+        return (
+            nextProps.events.length !== this.props.events.length ||
```

Comment:
@Kegsay said something along the lines of if the parent gets updated, then it will re-render each child component, but only if `shouldComponentUpdate` is true